spec/spec.md

```diff
@@ -203,6 +203,28 @@ This profile uses certain versions of specifications that have not yet reached f
   - First Implementer's Draft of OpenID for Verifiable Prensetations specification
   - ID Hub specification published as a v0.0.1 predraft of [Decentralized Web Node](https://identity.foundation/decentralized-web-node/spec/). We will continue to use the term ID Hub rather than Decentralized Web Node to avoid confusion.
 
+### JWT-VCs
+
+#### Using JWT claims instead of their counterparts in the data model specification
+
+Section 6.3.1 of [[ref: VC Data Model v1.1]] provides two options for how to encode properties defined in VC Data Model v1.1 as a JWT. 
+
+For the purpose of this profile, registered JWT claims `exp`, `iss`, `nbf`, `jti`, `sub` and `aud` MUST be used in a JWT-VC instead of their respective counterparts defined in VC Data Model v1.1.
```

Comment:
```suggestion
For the purpose of this profile, registered JWT claims `exp`, `iss`, `nbf`, `jti`, `sub` and `aud` MUST be used in a JWT VC instead of their respective counterparts defined in VC Data Model v1.1.
```